src/tracer/nw2_tracers.F90

```diff
@@ -0,0 +1,314 @@
+!> Ideal tracers designed to help diagnose a tracer diffusivity tensor in NeverWorld2
+module nw2_tracers
+
+! This file is part of MOM6. See LICENSE.md for the license.
+
+use MOM_diag_mediator, only : diag_ctrl
+use MOM_error_handler, only : MOM_error, FATAL, WARNING
+use MOM_file_parser, only : get_param, log_param, log_version, param_file_type
+use MOM_forcing_type, only : forcing
+use MOM_grid, only : ocean_grid_type
+use MOM_hor_index, only : hor_index_type
+use MOM_io, only : file_exists, MOM_read_data, slasher, vardesc, var_desc
+use MOM_restart, only : query_initialized, MOM_restart_CS
+use MOM_time_manager, only : time_type, time_type_to_real
+use MOM_tracer_registry, only : register_tracer, tracer_registry_type
+use MOM_tracer_diabatic, only : tracer_vertdiff, applyTracerBoundaryFluxesInOut
+use MOM_unit_scaling, only : unit_scale_type
+use MOM_variables, only : surface, thermo_var_ptrs
+use MOM_verticalGrid, only : verticalGrid_type
+
+implicit none ; private
+
+#include <MOM_memory.h>
+
+public register_nw2_tracers
+public initialize_nw2_tracers
+public nw2_tracer_column_physics
+public nw2_tracers_end
+
+!> The control structure for the nw2_tracers package
+type, public :: nw2_tracers_CS ; private
+  integer :: ntr = 0  !< The number of tracers that are actually used.
+  type(time_type), pointer :: Time => NULL() !< A pointer to the ocean model's clock.
+  type(tracer_registry_type), pointer :: tr_Reg => NULL() !< A pointer to the tracer registry
+  real, pointer :: tr(:,:,:,:) => NULL()   !< The array of tracers used in this package, in g m-3?
+  real, allocatable , dimension(:) :: restore_rate !< The exponential growth rate for restoration value [year-1].
```

Comment:
From the code, it appears that this damping rate is in [s-1]. not [year-1]. Also, to more fully implement the dimensional consistency testing and help in the detection of inconsistent expressions, I recommend that this be changed to [T-1 ~> s-1] by moving the conversion factor up to the point where restore_rate is first set.

Reply:
Yes, it is in [s-1]. This is a hold over from the other tracer code using [yr-1].

Reply:
Let us accept this PR as it currently stands, and then fix this very minor issue in a subsequent PR.